src/app/core/entity-components/entity-form/entity-form/entity-form.component.scss

```diff
@@ -21,6 +21,9 @@ $max-screen-width: $min-block-width + $margin-main-view-right + $margin-main-vie
 .columns-wrapper {
   @include adaptive-grid($min-block-width, $max-screen-width);
   align-items: start;
+  position: relative;
+  height: calc(100vh - 240px);
```

Comment:
because the 240px depend on the space taken up by our heading of the view above, this will stop working (i.e. start showing a second scroll bar) when we change things about other components outside of this.

The idea of CSS certainly is to lay out things well stacked and use relative measures ... but with all our components inside each other this is a huge pain in the a**. So let's go with this for now :wink:

Reply:
@Schottkyc137 should we consider reworking things from the root inwards, so that the overall views always exactly fill the screen and inside components like this one take up relative space and display their own scrolling?

Reply:
> @Schottkyc137 should we consider reworking things from the root inwards, so that the overall views always exactly fill the screen and inside components like this one take up relative space and display their own scrolling?

At some point, I think that this is the way to go. That being said, our large HTML tree certainly complicates this (because a lot of elements need to have their height set appropriately) but in the end, it might be worth it.